sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -959,6 +959,11 @@ def _add_argparse_args(cls, parser):
                         help='Flink version to use.')
     parser.add_argument('--flink_job_server_jar',
                         help='Path or URL to a flink jobserver jar.')
+    parser.add_argument('--flink_submit_uber_jar',
+                        default=False,
+                        action='store_true',
+                        help='Create and upload an uberjar to the flink master'
+                             ' directly, rather than starting up a job server.')
```

Comment:
An alternative could be to document that this flag can only be used with Python >= 3.6 and when `flink_master` is set to a cluster address.

Reply:
Added more detail to the help string.